File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        unique_list = list(dict.fromkeys(nums))
        print(unique_list)
        if len(unique_list) == len(nums):
            return False
        else:
            return True



       

File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Map<Integer, Integer> countHashMap = new HashMap<>();
        for (int num : nums) {
            if(countHashMap.containsKey(num)) {
                countHashMap.put(num, countHashMap.get(num)+1);
            }
            else {
                countHashMap.put(num, 1);
                 }
         }
         for (int count: countHashMap.values()) {
             if (count>1){
                 return true;
             }
         }
        return false;                 
     } 
}